wordpress-mobile/WordPress-iOS
# Use credentials tool to copy/decrypt credentials intead of referencing the secrets repo path

Currently, our credentials generation references the `~/.mobile-secrets` path directly. This is a problem for cases where the secrets repo doesn't exist (such as CI).

This is an update to use our existing credentials tool to copy/decrypt the relevant files within the repo. This will allow CI to decrypt these files if needed. It also simplifies the setup for third party contributors (no need to create a `~/.mobile-secrets` directory).

To test:

1. Run `rake dependencies`.
2. Confirm that `wpcom_alpha_app_credentials`, `wpcom_app_credentials` and `wpcom_internal_app_credentials` are present in the `WordPress/Credentials` directory.
3. Open the workspace in Xcode and build & run the app. Confirm WP.com login still works.

Update release notes:
